README.adoc

```diff
@@ -11,7 +11,7 @@ endif::[]
 https://learnteachcode.org/code-coffee-compendium/home/
 
 == Introduction
-Welcome to the LearnTeachCode Code & Coffee Compendium Github Repository! Maintained by the Baldwin Park & El Sereno/Alhambra study group, this repository hosts the source code for https://learnteachcode.org/code-coffee-compendium/[code-coffee-compendium] resource site. 
+Welcome to the LearnTeachCode Code & Coffee Compendium Github Repository! Maintained by the Baldwin Park & El Sereno/Alhambra study group, this repository hosts the source code for https://github.com/LearnTeachCode/code-coffee-compendium resource site. 
```

Comment:
We shouldn't change this link, since the README is primarily viewed via GitHub, so this link would just redirect to itself. If the intent is to close issue #90, then https://github.com/LearnTeachCode/code-coffee-compendium should go in [home.adoc](https://github.com/LearnTeachCode/code-coffee-compendium/blob/main/content/home.adoc)

Reply:
Oh okay whoops.  I'll change it the home.adoc file later.